 Description: reduce the rust debuginfo level
 because compiling with debuginfo=2 causes the OOM killer to interrupt the build
 on launchpad builders. Initially this was only on 32 bit architectures, but
 with firefox 63 it started happening frequently on arm64 and ppc64el too,
 with newer versions it started happening very frequently on s390x too, and with
 firefox 84 (built with rustc 1.47) it started happening on amd64 too.
 This patch would initially decrease debug_info for selected architectures, but
 with recent versions of rustc pretty much all supported architectures are
 affected, so it is now unconditional.

--- a/build/moz.configure/toolchain.configure
+++ b/build/moz.configure/toolchain.configure
@@ -2167,7 +2167,7 @@ def rust_compile_flags(opt_level, debug_
         debug_assertions = False
 
     if debug_symbols:
-        debug_info = "2"
+        debug_info = "1"
 
     opts = []
 
